pkg/sqlmigrations/migrations.go

```diff
@@ -1851,3 +1857,78 @@ func depublicizeSystemComments(ctx context.Context, r runner) error {
 	}
 	return nil
 }
+
+func renameColumnFamily(
```

Comment:
I'll happily defer to the opinions of the sql and sqlmigration experts of @lucy-zhang (and @jordanlewis), and I know I originally mentioned that maybe it'd be possible to resolve my concern by renaming, but seeing this actually worked out, personally, this feels like a lot more migration code and associated complexity/risk for the benefit of just a shorter name. ~nobody has to see / type this name except when adding columns to this table so I don't know if it is worth all this?